java/build.gradle

```diff
@@ -33,10 +33,10 @@ sourceSets {
 }
 
 dependencies {
-    compile "org.whispersystems:curve25519-java:${curve25519_version}"
-    compile 'com.google.protobuf:protobuf-javalite:3.10.0'
+    implementation "org.whispersystems:curve25519-java:${curve25519_version}"
+    implementation 'com.google.protobuf:protobuf-javalite:3.10.0'
 
-    testCompile ('junit:junit:3.8.2')
+    testImplementation ('junit:junit:3.8.2')
```

Comment:
Use of 'testCompile` has been deprecated in Gradle, replaced by `testImplementation`